stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AccessibilityNodeInfoWrapper.java

```diff
@@ -0,0 +1,65 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.support.v4.view.ViewCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.view.View;
+import android.view.ViewParent;
+
+public class AccessibilityNodeInfoWrapper {
```

Comment:
Can we make this into a class with static utility methods? I'm not sure we need an allocation for the sake of a `getIgnored` method with 3 parameters (or even for when/if we add more methods)